mw-webapp/src/logic/wayPage/reportsTable/ReportsTable.module.scss

```diff
@@ -0,0 +1,3 @@
+.positionTd {
+  position: relative;
```

Comment:
Why

Reply:
We need it for positioning plus  buttons in the cells